src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilter.java

```diff
@@ -60,6 +60,11 @@
      */
     private boolean optional;
 
+    /**
+     * Control if only consider added lines in file.
+     */
+    private boolean add;
```

Comment:
Please refer to the initial doc and call it `strategy`.

Reply:
done